tests/playwright/src/model/ai-lab-page.ts

```diff
@@ -33,4 +38,14 @@ export class AILabPage extends AILabBasePage {
     await playExpect(this.heading).toBeVisible();
     await this.navigationBar.waitForLoad();
   }
+
+  static async openAILabDashboard(runner: Runner, page: Page, navigationBar: NavigationBar): Promise<AILabPage> {
```

Comment:
this does not belongs here, this functionality operates outside of the ai-lab webview. Isn't it better in some specific file like utility or spec?

Reply:
resulting object is AILabPage, I wanted to extract the functionality somwehere from the spec file, this made most sense to me, don't see any benefit in splitting it up into a util class

Reply:
corrected